### ISSUE TITLE
Access tabbar in shadow DOM

### ISSUE THREAD

**Issue by georgechr** [ionic-team/ionic#15661]
Would it be possible to set a CSS variable for controlling width/height of tabbar inside shadow DOM?

**Comment by RHinderiks** [comment]
May i ask why you could want to increase or decrease the height of the tabbar ? and more puzzling the width ?

**Comment by georgechr** [comment]
The problem is that it sits inside shadow DOM for tabbar, otherwise we would have styled it as we wished

**Comment by RHinderiks** [comment]
right, thats annoying. I feel shadow dom is a huge improvement mainly for the people of ionic but as a developer it's quite restricting (which sometimes is a good thing) However i don't know enough about shadow dom so maybe their still is a way. Calling @mhartington and/or other ionic employees to take this thread over.

**Comment by mhartington** [comment]
👋 

Hey there! So we're actively exposing css properties for all the components as we release new betas. While it might not be exposed now, we'll probably add a variable for you access it as we get close to RC/final.

Currently we've only exposed certain properties in order to get Theming docs done and documenting the properties we do have. 

I'll add this to the backlog for now so we can track it.
